### PR TITLE
Page user

### PR DESCRIPTION
**Pull request by bdvorianov-sapient**

### CODE REVIEW

File: client/src/pages/chat/Chat.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    const navigate = useNavigate();
+    const to = (path) => navigate(path);
+
+    return(
+        <div className={s.chatWrapper}>
+            <div className={s.header}>
+                <div className={s.headerControls}>
+
+                    <button onClick={()=>to('/')}>
```
Comment (bdvorianov-sapient) [inline client/src/pages/chat/Chat.jsx:19]: this seems to be a link, isn't it?

File: client/src/pages/chat/Chat.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                            handlerClick={to}
+                            to={'/user/id'}/>
+
+                    <button>
+                        <img src={camera} alt="arrow"/>
+                    </button>
+
+                </div>
+
+                <h5 className={s.userName} onClick={()=>to('/user/id')}>Hell Boy</h5>
```
Comment (bdvorianov-sapient) [inline client/src/pages/chat/Chat.jsx:33]: same here, if you click on something and it leads you to another route - it's a link

File: client/src/pages/chat/Chat.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+
+            </div>
+
+            <div className={s.chatContent}>
+
+                <Message own={false} text={'Привет!'}/>
+                <Message own={true} text={'Говори по Українськи!'}/>
```
Comment (bdvorianov-sapient) [inline client/src/pages/chat/Chat.jsx:41]: 🤘🏻 🇺🇦

https://www.youtube.com/watch?v=x42oQ-MO4A8

File: client/src/pages/user/user.jsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+            <div className={s.userWrapper}>
+
+                <div className={s.userHeader}>
+                    <button onClick={()=>navigate('/chat')}>
+                        <span>Cancel</span>
+                    </button>
+
+                    <div className={s.userAvatar}>
+                        <Avatar url={'https://avochka.ru/img/kartinka/1/enot_glass.jpg'}
+                                style={{width: '100px', height: '100px'}}
```
Comment (bdvorianov-sapient) [inline client/src/pages/user/user.jsx:20]: stuff like this should be done by proper classes

File: client/src/pages/user/user.jsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+const User = ({}) =>{
+
+    const navigate = useNavigate();
+
+    return(
+        <>
+            <div className={s.userWrapper}>
+
+                <div className={s.userHeader}>
+                    <button onClick={()=>navigate('/chat')}>
```
Comment (bdvorianov-sapient) [inline client/src/pages/user/user.jsx:14]: Link 👀

File: client/src/pages/chat/Chat.jsx (hunk 1 above)
Comment (kutuzovpetia) [reply client/src/pages/chat/Chat.jsx]: I corrected

File: client/src/pages/chat/Chat.jsx (hunk 2 above)
Comment (kutuzovpetia) [reply client/src/pages/chat/Chat.jsx]: I corrected

File: client/src/pages/chat/Chat.jsx (hunk 3 above)
Comment (kutuzovpetia) [reply client/src/pages/chat/Chat.jsx:39]: 🤘🏻

File: client/src/pages/user/user.jsx (hunk 4 above)
Comment (kutuzovpetia) [reply client/src/pages/user/user.jsx]: I corrected

File: client/src/pages/user/user.jsx (hunk 5 above)
Comment (kutuzovpetia) [reply client/src/pages/user/user.jsx]: I corrected

File: client/src/pages/user/user.jsx [hunk 6]
```diff
@@ -3,30 +3,30 @@ import Avatar from '../../components/avatar';
...
                 </div>
+            </header>
 
+            <section>
                 <ul className={s.list}>
                     <li>Hell</li>
                     <li>Boy</li>
                 </ul>
-                <span className={s.context}>Enter your name adn add an optional profile photo</span>
+                <abbr className={s.context}>Enter your name adn add an optional profile photo</abbr>
```
Comment (bdvorianov-sapient) [inline client/src/pages/user/user.jsx:29]: abbr is not the one needed here

File: client/src/components/avatar/avatar.jsx [hunk 7]
```diff
@@ -1,17 +1,23 @@
...
+    let styleAvatar = null;
+    switch (size){
+        case 'small': styleAvatar = s.small;
+            break;
+        case 'medium': styleAvatar = s.medium;
+            break;
+        case 'large': styleAvatar = s.large;
+            break;
+        default: styleAvatar = s.small;
+    }
```
Comment (bdvorianov-sapient) [inline client/src/components/avatar/avatar.jsx:15]: this also can be solved with `classnames` library

check this out: https://www.npmjs.com/package/classnames

File: client/src/pages/chat/Chat.jsx [hunk 8]
```diff
@@ -2,36 +2,34 @@ import s from './chat-style.module.sass';
...
-    const navigate = useNavigate();
-    const to = (path) => navigate(path);
 
     return(
         <div className={s.chatWrapper}>
             <div className={s.header}>
                 <div className={s.headerControls}>
 
-                    <button onClick={()=>to('/')}>
+                    <a href="/">
```
Comment (bdvorianov-sapient) [inline client/src/pages/chat/Chat.jsx:14]: In this case you should use react-router link

File: client/src/pages/chat/Chat.jsx [hunk 9]
```diff
@@ -2,36 +2,34 @@ import s from './chat-style.module.sass';
...
                         <img src={arrow} alt="arrow"/>
-                    </button>
+                    </a>
 
-                    <Avatar url={'https://avochka.ru/img/kartinka/1/enot_glass.jpg'}
-                            handlerClick={to}
-                            to={'/user/id'}/>
+                    <a href="/user/id">
+                        <Avatar url={'https://avochka.ru/img/kartinka/1/enot_glass.jpg'} size={'medium'}/>
```
Comment (bdvorianov-sapient) [inline client/src/pages/chat/Chat.jsx:18]: url='https://www.parisbeacon.com/wp-content/uploads/2021/11/Rocket-Raccoon-James-Gunn-Guardianes-de-la-Galaxia-Marvel-Studios.jpg'

**Pull request merged**